web/angular-wallet/src/app/main/set-commitment/set-commitment.component.ts

```diff
@@ -0,0 +1,59 @@
+import {Component, OnInit} from '@angular/core';
+import {SuggestedFees, Account} from '@burstjs/core';
+import {ActivatedRoute} from '@angular/router';
+import {AccountService} from 'app/setup/account/account.service';
+import {StoreService} from '../../store/store.service';
+import {takeUntil} from 'rxjs/operators';
+import {getBalancesFromAccount} from '../../util/balance';
+import {UnsubscribeOnDestroy} from '../../util/UnsubscribeOnDestroy';
+
+// test
```

Comment:
test

Reply:
yeah... I had issues with the CI pipeline and was testing things... clean up will come.